src/pykiso/lib/auxiliaries/udsaux/uds_server_auxiliary.py

```diff
@@ -139,7 +139,9 @@ def send_response(self, response_data: List[int]) -> None:
         :param response_data: the UDS response to send.
         """
         to_send = self.uds_config.tp.encode_isotp(
-            response_data, use_external_snd_rcv_functions=True
+            response_data,
```

Comment:
As we discussed when testing, could you also please fix my bug by adding the `callback` parameter in `register_callback` (so l.196 and l.220)?

Reply:
Done